ls1intum/Artemis
# `General`: Prevent Exercise Group button from leaving view

### Checklist
#### General


#### Client

### Motivation and Context

It's an old issue that was labeled `good first issue` so I figured I have a go at it. Hope it's alright and if it's not needed anymore, just close the PR unmerged.

#5525 

### Description
I added the css class `table-responsive` to a new `div` that wraps the original table. This prevents the button from leaving the view. As a result the table is now more responsive and horizontally scrollable if the content does not fit.
| before | now |
|--------|--------|
| <img alt="before" src="https://github.com/ls1intum/Artemis/assets/16179620/5bc96f0f-e7af-4d10-a1ab-0cfa97bf20be"> | ![after](https://github.com/ls1intum/Artemis/assets/16179620/66251c90-128f-47d0-94fd-374a587885ff) |
### Steps for Testing

Prerequisites:
- 1 Exam

1. Go to the Exam Management
2. Click on Exam Checklist
3. Resize the view
4. Check if the button leaves the view

#### Exam Mode Testing



Prerequisites:
- 1 Exam

1. Go to the Exam Management
2. Click on Exam Checklist
3. Resize the view
4. Check if the button leaves the view

### Review Progress






#### Performance Review
#### Code Review
#### Manual Tests
#### Exam Mode Test

### Test Coverage







### Screenshots
